packages/theme-check-common/src/types/schemas/section.ts

```diff
@@ -0,0 +1,107 @@
+import { Setting } from './setting';
+
+export declare namespace Section {
+  /** {% schema %} */
+  export interface Schema {
+    name?: string;
+    tag?: 'article' | 'aside' | 'div' | 'footer' | 'header' | 'section';
+    class?: string;
+    limit?: number;
+    settings?: Setting.Any[];
+    max_blocks?: number;
+    blocks?: Block[];
+    presets?: Preset[];
+    default?: Default;
+    locales?: Record<string, Record<string, string>>;
+    enabled_on?: SectionToggle;
+    disabled_on?: SectionToggle;
+  }
```

Comment:
JSON Schema -> TypeScript translation thanks to Cursor. I have _briefly_ looked at everything and it seems correct at first glance.